### DISCUSSION TITLE
What does 'Extreme Poverty' mean?

### DISCUSSION THREAD

**Discussion by KrutarthSompura** [owid/covid-19-data#2772] [Q&A] [answered]
I would like to know what the Extreme Poverty column in the dataset means?

I compared numbers for few countries, but it did not make sense. 

**US: 1.2
UK: 0.2
India: 21.2
Pakistan: 4**

I infrered 'Extreme Poverty' as the 'Poverty Level' for each country. Based on above numbers, considering Extreme Poverty as Poverty Level, UK should have relatively higher number than US, and Pakistan should have a higher number than India. But it's the opposite here.

Hence, I would like to know what Extreme Poverty column excatly justify.

Thank you.

**Comment by lucasrodes** [comment]
Hi @KrutarthSompura,
For general questions about OWID articles and data (not necessarily COVID-related), please refer to our feedback form at https://ourworldindata.org/ (button on the bottom right).

In the meantime, you can read all our content on "Poverty" at https://ourworldindata.org/poverty.

Thanks

**Comment by edomt** [comment, marked as answer]
Hi @KrutarthSompura 

The column `extreme_poverty` is sourced from the World Bank World Development Indicators. It refers to the share of the population living in extreme poverty, for the most recent year available since 2010.

You can find more information about each variable in the dataset in our codebook [here](https://github.com/owid/covid-19-data/blob/master/public/data/owid-covid-codebook.csv).

**Reply by KrutarthSompura** [reply]
Thank you.
